spring-cloud-deployer-resource-maven/src/main/java/org/springframework/cloud/deployer/resource/maven/MavenProperties.java

```diff
@@ -41,10 +43,13 @@
 	 */
 	private String localRepository = DEFAULT_LOCAL_REPO;
 
+	private static final String DEFAULT_REMOTE_REPO_KEY = "default" + new Random().nextInt();
+
 	/**
 	 * Locations of remote maven repositories from which artifacts will be downloaded, if not available locally.
 	 */
-	private Map<String, RemoteRepository> remoteRepositories = new HashMap<>();
+	private Map<String, RemoteRepository> remoteRepositories = new HashMap<>(
+			Collections.singletonMap(DEFAULT_REMOTE_REPO_KEY, new RemoteRepository("http://repo1.maven.org/maven2")));
```

Comment:
Since having direct singleton map here makes it `immutable`, creating a new `HashMap`.